ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/headervalidationrules/ProofOfWorkValidationRule.java

```diff
@@ -47,8 +47,10 @@ public boolean validate(final BlockHeader header, final BlockHeader parent) {
     }
     final BigInteger difficulty =
         BytesValues.asUnsignedBigInteger(header.getDifficulty().getBytes());
-    final UInt256 target = UInt256.of(ETHHASH_TARGET_UPPER_BOUND.divide(difficulty));
-
+    final UInt256 target =
```

Comment:
What do you think about creating a new `ethash` package somewhere (ethereum/core/.../mainnet/ethash??) to consolidate these types of utilities?

Reply:
Sounds like a good idea, probably as part of ProgPow work.  Opened as PAN-2998